downstream/modules/troubleshooting-aap/proc-troubleshoot-job-timeout.adoc

```diff
@@ -0,0 +1,54 @@
+[id="troubleshoot-job-timeout"]
+= Issue - Jobs are failing with “Timeout (12s) waiting for privilege escalation prompt” error message
+
+This error can happen when the timeout value is too small, causing the job to stop before completion. The default timeout value for connection plugins is `10`. 
+
+To resolve the issue, increase the timeout value by completing one of the following procedures. 
+
+[NOTE]
+====
+The following changes will affect all of the jobs in {ControllerName}. To use a timeout value for a specific project, add an `ansible.cfg` file in the root of the project directory and add the `timeout` parameter value to that `ansible.cfg` file.
+====
+
+*Add ANSIBLE_TIMEOUT as an environment variable in the {ControllerName} UI*
+
+. Go to {ControllerName}.
+. From the navigation panel, select menu:Settings[Jobs settings].
+. Under *Extra Environment Variables* add the following:
++
+----
+{ 
+"ANSIBLE_TIMEOUT": 60 
+}
+----
++
+
+
+*Add a timeout value in the [defaults] section of the ansible.cfg file by using the CLI*
+
+* Edit the `/etc/ansible/ansible.cfg` file and add the following:
++
+----
+[defaults]
+timeout = 60
+----
++
+
+
+*Running ad hoc commands with a timeout*
+
+* To run an ad hoc playbook in the command line, add the `--timeout` flag to the `ansible-playbook` command, for example:
++
+[subs="+quotes"]
+----
+# ansible-playbook --timeout=60 _<your_playbook.yml>_
+----
++
+
+
+[role="_additional-resources"]
+.Additional resources
```

Comment:
I believe we're supposed to format additional resources sections like this: == Additional resources. This is so that asciidoc recognizes it as a sub heading.

Reply:
Thanks for flagging this 👍  Will double check with Donna as I remember her sharing something about this formatting a while ago (perhaps that advice is outdated now).